articles/getting-started-with-operator-overloading-in-c++/index.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/getting-started-with-operator-overloading-in-c++/
+title: Getting Started with Operator Overloading in C++
+description:
+author: dawe-daniel
+date: 2020-12-01T00:00:00-17:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/getting-started-with-operator-overloading-in-c++/hero.jpg
+    alt: example image
+
+---
+Operator overloading is the process of making an operator exhibit different behaviors in different instances. By overloading operators in a specific class, you can the user's view of that class. This helps developers have a problem driven approach towards programming.
```

Comment:
@Dawe-7 

Quick ? 

Im going through my final review but im noticing this sentence may be missing a word:

"By overloading operators in a specific class, you can the user's view of that class. This helps developers have a problem driven approach towards programming.''

Think we're missing a word after "you can" blank the user's view? 

Im assuming its change? Change the users view - but I wanted to double check. 
👍  
Thank you

Reply:
@hectorkambow,
Yes, it's " you can change the user's view". sorry about that.

Reply:
@hectorkambow,
I have made the change. Please review